lib/myaccount/request.ts

```diff
@@ -26,23 +27,44 @@ export async function sendRequest<
   const {
     accessToken: accessTokenObj
   } = oktaAuth.tokenManager.getTokensSync();
-  
-  const accessToken = options.accessToken || accessTokenObj?.accessToken;
+
+  const atToken = options.accessToken || accessTokenObj;
   const issuer = oktaAuth.getIssuerOrigin();
   const { url, method, payload } = options;
   const requestUrl = url.startsWith(issuer!) ? url : `${issuer}${url}`;
 
-  if (!accessToken) {
+  if (!atToken) {
     throw new AuthSdkError('AccessToken is required to request MyAccount API endpoints.');
   }
-  
-  const res = await httpRequest(oktaAuth, {
+
+  let accessToken: string | AccessToken = atToken;
```

Comment:
Curious why we need to create a new variable here instead of reusing `atToken`

Reply:
Same as above